_includes/v19.1/app/txn-sample.clj

```diff
@@ -3,8 +3,14 @@
             [test.util :as util]))
 
 ;; Define the connection parameters to the cluster.
-(def db-spec {:subprotocol "postgresql"
-              :subname "//localhost:26257/bank"
+(def db-spec {:dbtype "postgresql"
```

Comment:
I'm deducing that the reason `:dbtype` and friends were not used here is that perhaps something didn't work in sekuritay-land did not like `:subprotocol`, etc.?  (I'm being too lazy to find the answer myself, sorry)

Reply:
I tried out using `:dbtype` and friends and they worked for the insecure example! I'll change this to make it standard across the insecure and secure examples.